diff --git a/cmake/opentelemetry-proto.cmake b/cmake/opentelemetry-proto.cmake
index a21c0f1..5115b9d 100644
--- a/cmake/opentelemetry-proto.cmake
+++ b/cmake/opentelemetry-proto.cmake
@@ -272,13 +272,9 @@ install(
   FILES_MATCHING
   PATTERN "*.h")
 
-if(TARGET protobuf::libprotobuf)
-  target_link_libraries(opentelemetry_proto PUBLIC protobuf::libprotobuf)
-else() # cmake 3.8 or lower
-  target_include_directories(opentelemetry_proto
-                             PUBLIC ${Protobuf_INCLUDE_DIRS})
-  target_link_libraries(opentelemetry_proto INTERFACE ${Protobuf_LIBRARIES})
-endif()
+target_include_directories(opentelemetry_proto
+                           PUBLIC ${Protobuf_INCLUDE_DIRS})
+target_link_libraries(opentelemetry_proto PUBLIC ${Protobuf_LIBRARIES})
 
 if(BUILD_SHARED_LIBS)
   set_property(TARGET opentelemetry_proto PROPERTY POSITION_INDEPENDENT_CODE ON)
